HunterPie.Core/Game/Rise/MHRGame.cs

```diff
@@ -81,6 +82,19 @@ private set
         }
     }
 
+    public int MaxDeaths
+    {
+        get => _maxDeaths;
+        private set
+        {
+            if (value != _maxDeaths)
+            {
+                _maxDeaths = value;
+                this.Dispatch(OnDeathCountChange, this);
+            }
+        }
+    }
+
```

Comment:
Like I said in the previous review, there's no need to dispatch events for `Max____` properties

```suggestion
    public int MaxDeaths { get; private set; }
```